ghost/core/core/server/data/schema/fixtures/fixtures.json

```diff
@@ -763,7 +768,8 @@
                     "newsletter": "all",
                     "explore": "read",
                     "comment": "all",
-                    "link": "all"
+                    "link": "all",
+                    "mention": "browse"
```

Comment:
We need to add the same to `"Admin Integration": {` a couple of lines further. Just like the test fixtures one